README.md

```diff
@@ -39,8 +39,8 @@ No requirements.
 
 | Name | Version |
 |------|---------|
-| <a name="provider_google"></a> [google](#provider\_google) | n/a |
-| <a name="provider_google-beta"></a> [google-beta](#provider\_google-beta) | n/a |
+| <a name="provider_google"></a> [google](#provider\_google) | 4.47.0 |
```

Comment:
you might have to manually revert this to n/a, and same below.

when this runs on gha, it doesnt have the provider installed, so it shows up as n/a.

Reply:
done